src/ucp/core/ucp_worker.c

```diff
@@ -3177,7 +3175,8 @@ static int ucp_worker_do_ep_keepalive(ucp_worker_h worker, ucs_time_t now)
     ucs_trace("ep %p: do keepalive on lane[%d]=%p ep->flags=0x%x", ep, lane,
               ep->uct_eps[lane], ep->flags);
 
-    if (ucp_ep_is_am_keepalive(ep, rsc_index, is_p2p)) {
+    if (ucp_ep_is_am_keepalive(
+                ep, rsc_index, ucp_ep_config(ep)->p2p_lanes & UCS_BIT(lane))) {
```

Comment:
this wrapping seems weird, is it from clang-format?

Reply:
fixed